.travis.yml

```diff
@@ -8,7 +8,12 @@ before_install:
   - sudo apt-get install -y librsvg2-bin npm
 
 install:
-  - pip install .[test]
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      pip install .[test] cairosvg==1.0.22 pillow;
+    else
+      pip install .[test] cairosvg pillow;
+    fi
+
```

Comment:
Can you please move this test to setup.py instead, like this:

```
diff --git a/setup.py b/setup.py
index eb7dd4a..62e4301 100644
--- a/setup.py
+++ b/setup.py
@@ -5,6 +5,8 @@ https://github.com/pypa/sampleproject
 """
 
 # Always prefer setuptools over distutils
+import sys
+
 from setuptools import setup, find_packages
 # To use a consistent encoding
 from codecs import open
@@ -142,7 +144,11 @@ setup(
     # projects.
     extras_require={  # Optional
         # "dev": ["check-manifest"],
-        "test": ["xmldiff"],
+        "test": [
+            "xmldiff",
+            "cairosvg==1.0.22" if sys.version_info <= (3, ) else "cairosvg",
+            "pillow"
+        ],
     },
 
     setup_requires=[
```

Reply:
I wasn't even aware that was possible.  That's a much better place.  I'll amend that commit tonight.

Reply:
Done, except I changed `sys.version_info <= (3, )` to `sys.version_info < (3, )`.  The head version of *cairosvg* as of now should work with 3.0 and up, and your edit would cause cairosvg==1.0.22 to be used for Python 2.7 and 3.0, but not 3.1 and up.

I also removed the addition of Python 3.7 to the test environments:  That isn't really relevant to functionality this pull request adds.  I will start a separate Issue to discuss the test environments.